apps/barista-design-system/src/components/example.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Compiler,
+  Component,
+  Input,
+  NgModule,
+  OnInit,
+  Type,
+} from '@angular/core';
+import { from, Observable } from 'rxjs';
+import { filter, map, switchMap } from 'rxjs/operators';
+
+@Component({
+  selector: 'ba-live-example',
+  template: `
+    <ng-container [ngComponentOutlet]="example$ | async"></ng-container>
+  `,
+})
+export class BaLiveExample implements OnInit {
+  /** The name of the example (class name) that will be instantiated. */
+  @Input() name: string;
+
+  example$: Observable<any>;
+
+  constructor(private _compiler: Compiler) {}
+
+  ngOnInit(): void {
+    this._initExample();
+  }
+
+  private _initExample(): void {
+    this.example$ = from(import(`@dynatrace/examples/chart`)).pipe(
```

Comment:
Should this be always the `examples/chart`?

Reply:
this file is garbage will be deleted.